main.cpp

```diff
@@ -5,14 +5,17 @@
 // #include <cstring>
 // #include <dlfcn.h>
 // #include <errno.h>
+#include <iostream>
 #include <string>
 #include <map>
 #include <thread>
 #include <v8.h>
-#include <ml_logging.h>
```

Comment:
Create a seperate .cpp for android and ML?

Reply:
I think we could have a shared main entry point for mobiles. This was recommended here: https://github.com/webmixedreality/exokit/pull/889#issuecomment-480064282

This can be effected via `#define` -- and should definitely be done before merging this PR.